openstudiocore/CMakeLists.txt

```diff
@@ -12,6 +12,34 @@ endif()
 
 project(OpenStudio VERSION 2.7.2)
 
+# conan
+
+# Download automatically, you can also just copy the conan.cmake file
+if(NOT EXISTS "${CMAKE_BINARY_DIR}/conan.cmake")
+  message(STATUS "Downloading conan.cmake from https://github.com/conan-io/cmake-conan")
+  file(DOWNLOAD "https://github.com/conan-io/cmake-conan/raw/v0.13/conan.cmake"
+    "${CMAKE_BINARY_DIR}/conan.cmake")
+endif()
+
+include(${CMAKE_BINARY_DIR}/conan.cmake)
+
+message("RUNNING CONAN")
+conan_check(VERSION 1.0.0 REQUIRED)
+
+conan_add_remote(NAME bincrafters
+  URL https://api.bintray.com/conan/bincrafters/public-conan)
+
+conan_cmake_run(REQUIRES OpenSSL/1.1.0i@conan/stable
```

Comment:
What is the best place to find this recipe?  I searched around on bintray, but didn't find the recipe to see what options there are etc.

Reply:
It seems like there is a lot going on in this command, it is replacing three of our dependencies.....

Reply:
What does the `BUILD missing` option mean?  Where are we specifying static libs?